src/app/services/query.service.ts

```diff
@@ -338,6 +339,8 @@ export class QueryService {
               this.isUpdating_2 = false;
               this.isDirty_2 = false;
               this.isDirty_3 = true;
+              // update the export variables
+              this.exportService.updateExports();
```

Comment:
previously the updateExports seems not to be here as well, should be in update_2?